src/PrintfulProducts.php

```diff
@@ -44,11 +44,13 @@ public function __construct(PrintfulApiClient $printfulClient)
      * @throws Exceptions\PrintfulApiException
      * @throws Exceptions\PrintfulException
      */
-    public function getProducts($offset = 0, $limit = 20)
+    public function getProducts($status = 'all', $search = '', $offset = 0, $limit = 20)
```

Comment:
Add new parameters at the end of the list, as this would break every implementation that passes `$offset` as the first parameter